src/main/webapp/mentors.js

```diff
@@ -50,6 +50,7 @@ function getMentors() {
       console.log(mentor);
       mentorsContainer.appendChild(createMentorElement(mentor));
     }
+    mailtouiApp.run();
```

Comment:
The linter doesn't seem to like this -- you might need to add it to the list of globals in the `.eslintrc` file (I'll probably have to do the same on my PR)